joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/passes/RubyImportResolverPass.scala

```diff
@@ -44,6 +43,7 @@ class RubyImportResolverPass(cpg: Cpg) extends XImportResolverPass(cpg) {
       val resolvedTypeDecls = cpg.typeDecl
         .where(_.file.name(filePattern))
         .fullName
+        .filter(!_.endsWith(Defines.Program))
```

Comment:
Things that end with `:program` should have the `MODULE` modifier (including type decls), so this can be a `whereNot(_.isModule)` instead